Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml

```diff
@@ -50,14 +50,18 @@
   </display>
 
   <readouts>
+    <readout name="HCalBarrelReadout">
+      <!-- Introduce a segmentation in phi to devide HCAL modules, check for advantages of finer granularity -->
+     <id>system:4,module:8,row:9,layer:5</id>
+    </readout>
     <readout name="BarHCal_Readout">
       <!-- Introduce a segmentation in phi to devide HCAL modules, check for advantages of finer granularity -->
      <segmentation type="FCCSWGridPhiEta" grid_size_eta="100" phi_bins="256" offset_phi="-pi+(pi/256.)"/>
      <id>system:4,module:7,row:9,layer:5,tile:2,eta:1,phi:10</id>
     </readout>
-    <readout name="BarHCal_Readout_phieta">
-     <segmentation type="FCCSWGridPhiEta" grid_size_eta="0.01" offset_eta="-1.256" phi_bins="256" offset_phi="-pi+(pi/256.)"/>
-     <id>system:4,module:7,row:9,layer:5,tile:2,eta:9,phi:10</id>
+    <readout name="BarHCal_Readout_eta">
```

Comment:
If I remember correctly, `row` indicates the position in `z`, so why do you need to introduce this readout and `eta` segmentation?
Is this readout used somewhere?